changelog/702.misc.rst

```diff
@@ -0,0 +1 @@
+Limit installation of ``cchardet`` in the ``[speed]`` extra to Python versions below 3.10.
```

Comment:
We should link the aiohttp issue here to explain why.

Reply:
https://github.com/DisnakeDev/disnake/pull/702/commits/b91f5f67ebe44e4d45648c039da7d394faae42f1